### PR TITLE
Fixup various warnings from Bikeshed.

### PR DESCRIPTION
**Pull request by dj2**
This PR fixes most of the warnings generated by Bikeshed. The only
remaining warning relates to unused Biblio entries. The entries are
used, they're just used with an attached anchor and that doesn't seem to
trigger the Biblio usage check.

***
<a href="https://pr-preview.s3.amazonaws.com/dj2/FedCM/pull/123.html" title="Last updated on Nov 2, 2021, 3:23 AM UTC (eef6264)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/FedCM/123/a3cc440...dj2:eef6264.html" title="Last updated on Nov 2, 2021, 3:23 AM UTC (eef6264)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -384,39 +388,41 @@ to become [=directed identifiers=].
 ::
 
 <!-- ============================================================ -->
-# High Level Design # {#lifecycle}
+# High Level Design # {#high-level-design}
```
Comment (samuelgoto) [inline spec/index.bs:391]: thanks!

File: spec/index.bs [hunk 2]
```diff
@@ -801,6 +809,7 @@ enum FederatedCredentialApprovedBy {
 
 [Exposed=Window, SecureContext]
 partial interface FederatedCredential : Credential {
+  readonly attribute USVString id;
```
[the remark this answers is not in the document]
Comment (samuelgoto) [reply spec/index.bs:812]: I think the latter. The reference was for its parent's reference, not an overloading.

**Review by samuelgoto — COMMENTED** [review]
LGTM module minor comments. Will merge when responded to.

File: spec/index.bs [hunk 3]
```diff
@@ -78,12 +88,6 @@ The challenge is that some of these low level primitives are getting increasingl
 
 This specification defines a set of [=high-level API=]s that serve as a foundation where identity federation on Web can thrive without incurring into [[UNSANCTIONED-TRACKING]].
 
-<div class="example">
-  <pre class=include-raw>
-  path: ../explainer/static/mock46.svg
```
[the remark this answers is not in the document]
Comment (dj2) [reply spec/index.bs:83]: I ran into 2 issues:

1- it didnt' seem to make sense here
2- it's included elsewhere in the file, and bikeshed interprets a bunch of the ids in it to be duplicated.

File: spec/index.bs (hunk 2 above)
[the remark this answers is not in the document]
Comment (dj2) [reply spec/index.bs]: done.

**Comment by dj2** [comment]
Note, the build bot will be broken until #130 lands.

**Comment by dj2** [comment]
I believe this is now good to go.

**Pull request merged**